src/little-bang-parser/little_bang_generated_parser.mly

```diff
@@ -30,6 +31,7 @@ open Tiny_bang_parser_support;;
 %nonassoc EQUALITY LESS_THAN
 (* %right '<-' *)
 (* %left '+' '-' *)
+%nonassoc KEYWORD_REF
```

Comment:
Should `ref x = y` be parsed as:

1. `(ref x) = y`
2. `ref (x = y)`

?

I'm in favor of *2.*, it seems more natural. Currently, the code is doing *1.*, if I understand correctly.